examples/calendar-sensor.yaml

```diff
@@ -16,7 +16,7 @@ spec:
         group: argoproj.io
         version: v1alpha1
         kind: Workflow
-        artifactLocation:
+        source:
```

Comment:
+1 for changing this. Argo workflows themselves have a notion of artifacts and overloading that term was making it confusing.

Reply:
In a following change, we should also rename structs from `S3Artifact` to `S3Source` or something like that.

Reply:
agreed